docs/data-sources/pipeline.md

```diff
@@ -0,0 +1,26 @@
+# Data Source: pipeline
```

Comment:
Ohhh, I was curious how this works. Does Terraform scrape our repo for these docs to fill over at https://registry.terraform.io/providers/buildkite/buildkite/latest/docs?

Reply:
I believe so, but I have nfi how it works!

I assume the online docs only update when the terraform registry is notified about a new release?